app/LanguageNoticeWrapper.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+
+import LanguageNotice, { languageNoticePropType } from "../composites/Plugin/Shared/components/LanguageNotice.js";
+
+export const LanguageNoticeContainer = styled.div`
+	width: 100%;
```

Comment:
div elements are block elements, they already take all the available width. this seems unnecessary, unless I'm missing something